sm_project/cl_app/views.py

```diff
@@ -93,6 +93,24 @@ def add_item(request, checklist_id):
     else:
         return redirect('user_app:login')
 
+"""Removes a checklist item of the checklist with the corresponding id.
+
+Parameters
+----------
+checklist_id: int
+    The id of the checklist
+"""  
+def remove_item(request, checklistitem_id):
```

Comment:
You might be interested in checking out Django's `login_required` decorator. This will take care of the functionality to check the user is authenticated and redirect them to the login page if not for you (saving you some lines of code): https://docs.djangoproject.com/en/4.1/topics/auth/default/#django.contrib.auth.decorators.login_required